scripts/send-email-to-unresolved-breach-subscribers.js

```diff
@@ -38,18 +48,41 @@ async function sendUnresolvedBreachEmails (subscribers = null) {
     }
   }
 
-  await DB.destroyConnection()
-  console.log(`- Successfully emailed ${count} of ${subscribers.length} subscribers`)
+  console.log(`- Successfully emailed ${count} of ${subscribers.length} subscribers (${total} total)`)
+  return {
+    emailed: count,
+    attempted: subscribers.length,
+    total
+  }
 }
 
-if (argv[2]) {
-  // For testing, pass a comma-separated arg with no spaces, for example:
-  // node scripts/send-email-to-unresolved-breach-subscribers.js test1@test.com,test2@test.com,...
-  const subscribers = []
-  const emails = argv[2].split(',')
+async function init () {
+  LocaleUtils.init()
+  await EmailUtils.init()
+  await DB.createConnection()
+}
 
-  console.log('Testing job for monthly unresolved breach emails...')
-  console.log('- sending test emails to', emails)
+async function runScript (argv = []) {
+  let subscribers = null
+  const limit = AppConstants.MONTHLY_CRON_LIMIT
```

Comment:
I think anything in `.env` is returned as a string – probably need this to be `parseInt(AppConstants.MONTHLY_CRON_LIMIT)`?